src/main/java/com/github/onsdigital/babbage/template/handlebars/helpers/StringHelper.java

```diff
@@ -99,5 +102,28 @@ public CharSequence apply(String context, Options options) throws IOException {
         public void register(Handlebars handlebars) {
             handlebars.registerHelper(this.name(), this);
         }
-    }
+    },
+
+    wordCount {
+        // HTML tag regex pattern
+        private final Pattern pattern = Pattern.compile("\\<[^>]*>");
```

Comment:
I think we could make the comment here redundant by renaming the variable to reflect what the pattern is looking to match

Reply:
edited regex pattern name to better show function